lib/platform/github/index.ts

```diff
@@ -985,6 +986,10 @@ export async function setBranchStatus({
 
 /* istanbul ignore next */
 async function getIssues(): Promise<Issue[]> {
+  // istanbul ignore if
```

Comment:
Isn't this function fully ignored? See comment before declaration. 🤔

Did you checked if we need all those ignores?

Reply:
I attempted to get the initRepo mocking right and gave up..